tests/syscalls/test_ec.c

```diff
@@ -122,9 +178,75 @@ void test_scalar_mult_ec25519(void **state __attribute__((unused))) {
   assert_memory_equal(Pxy, expected, sizeof(expected));
 }
 
+static void test_eddsa_get_public_key(cx_curve_t curve, cx_md_t md,
+                                      const eddsa_test_vector *tv, size_t tv_len)
+{
+  cx_ecfp_512_private_key_t private_key;
+  cx_ecfp_512_public_key_t pub;
+  const cx_curve_domain_t *domain = cx_ecfp_get_domain(curve);
+  assert_non_null(domain);
+  size_t mpi_size = domain->length;
+
+  uint8_t secret_key[57];
+  uint8_t expected_key[114];
+  size_t i;
+
+  for (i = 0; i < tv_len; i++) {
+    assert_int_equal(hexstr2bin(tv[i].secret_key, secret_key, mpi_size), mpi_size);
+    assert_int_equal(hexstr2bin(tv[i].public_key, expected_key, mpi_size), mpi_size);
+
+    assert_int_equal(sys_cx_ecfp_init_private_key(curve, secret_key, mpi_size,
+                                              (cx_ecfp_private_key_t *)&private_key), mpi_size);
+
+    sys_cx_eddsa_get_public_key((cx_ecfp_private_key_t *)&private_key, md, (cx_ecfp_public_key_t *)&pub);
+    assert_int_equal(pub.W_len, 1 + 2 * mpi_size);
+    assert_int_equal(pub.W[0], 4);
+    sys_cx_edward_compress_point(curve, (uint8_t *) &pub.W, pub.W_len);
+
+    assert_int_equal(pub.W[0], 2);
+    assert_memory_equal(expected_key, &pub.W[1], mpi_size);
+  }
+}
+
+static void test_ed25519_get_public_key(void **state) {
+  (void)state;
```

Comment:
In another function of this file, there is an `unused` attribute instead: `void test_scalar_mult_ec25519(void **state __attribute__((unused))) {`.
It might be better to be consistent and use attributes everywhere or `(void)state;` everywhere, but not both.